fusor-ember-cli/app/styles/custom.scss

```diff
@@ -323,561 +292,9 @@ li.disabled a {
-  top: 15px;
-}
-.deployment-roles li.role-assigned .bootstrap-select {
-  display: inline-block !important;
-  margin-bottom: 4px;
-  width: auto !important;
-}
-.deployment-roles li.role-assigned .deployment-role-label {
-  color: #fff;
-}
-.deployment-roles li.role-assigned .role-option {
-  color: #fff;
-}
-.deployment-roles li.role-assigned .role-option:hover {
-  color: #e6e6e6;
-}
-.deployment-roles .role-block {
-  border-color: #409d35;
-}
-.deployment-roles li.role-block:before {
-  background-image: linear-gradient(to bottom, #409d35 60%, #ffffff 0%);
-  border-color: #409d35;
-}
-.deployment-roles li.role-block.role-assigned {
-  background: #92d400;
-}
-.deployment-roles .role-compute {
-  border-color: #ea7a23;
-}
-.deployment-roles li.role-compute:before {
-  background-image: linear-gradient(to bottom, #ea7a23 60%, #ffffff 0%);
-  border-color: #ea7a23;
-}
-.deployment-roles li.role-compute.role-assigned {
-  background: #f3a928;
-}
-.deployment-roles .role-controller {
-  border-color: #0d6f9a;
-}
-.deployment-roles .role-controller:before {
-  background-image: linear-gradient(to bottom, #0d6f9a 60%, #ffffff 0%);
-  border-color: #0d6f9a;
-}
-.deployment-roles .role-controller.role-assigned {
-  background: #1ba9df;
-}
-.deployment-roles .role-object {
-  border-color: #a2030d;
-}
-.deployment-roles .role-hidden {
-  visibility: hidden;
-}
-.deployment-roles li.role-object:before {
-  background-image: linear-gradient(to bottom, #a2030d 60%, #ffffff 0%);
-  border-color: #a2030d;
-}
-.deployment-roles li.role-object.role-assigned {
-  background: #ec4853;
-}
-.deployment-roles li.role-target {
-  border-style: dashed;
-  margin: 0;
-  margin-bottom: 10px;
-}
-.deployment-roles li.role-target:before {
-  display: none;
-}
-.deployment-roles li.role-target.role-target-active {
-  border-color: #7b7b7b;
-}
-.deployment-roles li.role-target.role-target-hover {
-  border-color: #3b3b3b;
-}
-.deployment-roles li.role-target > a {
-  display: block;
-  text-align: center;
-}
-.deployment-roles li.role-unconfigured {
-  background-color: #f8f8f8 !important;
-  border-color: #d1d1d1 !important;
-  color: #969696 !important;
-  cursor: auto;
-}
-.deployment-roles li.role-unconfigured:before {
-  display: none;
-}
-.deployment-roles li.ui-draggable:hover {
-  cursor: move;
-}
-.deployment-roles li .bootstrap-select {
-  display: none !important;
-}
-.deployment-roles li .pficon-help {
-  cursor: pointer;
-  font-size: 14px;
-  position: absolute;
-  left: 12px;
-  top: 10px;
-}
-.deployment-roles li .pficon-help:hover {
-  color: #555555;
-}
-.deployment-roles li .role-option {
-  display: none;
-  font-size: 14px;
-  position: absolute;
-  right: 10px;
-}
-.deployment-roles li .role-option.delete {
-  top: -2px;
-}
-.deployment-roles li .role-option.edit {
-  top: 4px;
-}
-.deployment-roles .draggable-object {
-  padding: 0;
-}
-.deployment-roles-assigned .draggable-object {
-  padding: 0;
-  display: block;
-}
-.roles-menu-item {
```

Comment:
@jeff-phillips-18, deployment-roles classes are all already in `openstack.scss`.

I compared the two files (`custom.scss` and `openstack.scss`) and there are only a few small differences and I assumed that `openstack.scss`is more up-to-date. Can you verify the differences and confirm/edit what is in openstack.css